### PR TITLE
Fix static.url call to stop it from printing HTML

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd - platform fix for static.url

### CODE REVIEW

File: lms/templates/courseware/courses.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
   if self.is_non_microsite_theme():
     course_index_overlay_text = _("Explore free courses from {university_name}.").format(university_name="Stanford University")
-    logo_file = static.url('themes/stanford/images/seal.png')
+    logo_file = settings.STATIC_URL + 'themes/stanford/images/seal.png'
```
Comment (stvstnfrd) [inline lms/templates/courseware/courses.html]: Did you say this is how edX does it elsewhere?

Comment (caesar2164) [inline lms/templates/courseware/courses.html]: https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/courses.html#L22

Comment (caesar2164) [inline lms/templates/courseware/courses.html]: btw, we actually have that comment on OUR line 22...

**Comment by stvstnfrd** [comment]
Cherry-picked at c6e14b863f985b3d59464c3f03bddff49d1715fb

**Pull request closed without merging**